technical/validations.md

```diff
@@ -7,20 +7,35 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
+- The name of the playground is between 3 en 40 characters ( no whitespaces in the beginning and trailing of the playground name )
```

Comment:
The no-white spaces statement can be a separate statement / validation. Also I think this is not a validation, but a processing requirement. The service can just trim the string and then do the validations. Or not, should we leave this to the front-end?

Reply:
Since AXON processes the commands and Events in an async way, It's good to validate the request before the event sourcing. If we store invalid events, then it never executes successfully irrespective of how many times its been replayed.

Considering back-end exposes an API, its good to validate all input data of a request.